client/app/components/username.tsx

```diff
@@ -17,6 +17,10 @@ function Username({ showVerfied, user }: Props) {
 			{user.username === "notgr" && (
 				<div className="inline-block i-lucide-crown text-amber-500" />
 			)}
+			{/* This should be for moderators */}
+			{user.role === "moderator" && user.username !== "notgr" && (
+				<div className="inline-block i-lucide-gem text-amber-500" />
```

Comment:
hmm gem? 

what about ghost or zap.

Reply:
And let's use a different color.

Reply:
I'm going with the zap in red-500. The lucide ghost just doesn't bang

Check it out
![image](https://github.com/blackmann/compa/assets/151869774/2addc316-4d47-4cc2-bac7-dc3bd6afc506)![image](https://github.com/blackmann/compa/assets/151869774/412b4f0c-7fda-4253-897d-b9130b68dc91)

Reply:
Lemme know what you think? Then I'll commit

Reply:
Looks cool... I like this too